docs: Update Readme (#47)
screwdriver-cd/sd-cmd: README.md

```

### Execute

> We execute a command. The command arguments can be specified by the following `arguments`.

Reviewer: ```suggestion
Executing a published command. The arguments for published command can be specified by the following `arguments`.
```

```bash
USAGE:

>    sd-cmd exec [command options] namespace/name@version [arguments...]

Reviewer 2: ```suggestion
   sd-cmd exec [options] namespace/name@version [arguments...]
```

- Use `-debug` of `--debug` option
- Set `SD_CMD_DEBUG_LOG` environment variable to `true`

### Publish

> We publish the command specified by a yaml.

Reviewer: ```suggestion
Publishing a command specified by a yaml.
```

```bash
USAGE:
   sd-cmd validate [command options]

OPTIONS:

>    -f, --f    Specify the path of yaml to validate (default: sd-command.yaml)

Reviewer 2: ```suggestion
   -f, --f string    Specify the path of yaml to validate (default: sd-command.yaml)
```

   -t, --t    Specify the tag given to the command (default: sd-command.yaml)

EXAMPLE:
   sd-cmd publish -f ./sd-command.yaml -t latest
```

### Validate

> We validate that a yaml is in the correct format for the command.

Reviewer: ```suggestion
Validating if a yaml is correct.
```

Author: [IMO] This seems the `validate` only checks the yaml is in correct format as yaml.

Reviewer: Is it better?
Validating if yaml is correct as sd-cmd format.

Author: It looks better for me.

USAGE:
   sd-cmd publish [command options]

OPTIONS:
   -f, --f    Specify the path of yaml to publish (default: sd-command.yaml)

>    -t, --t    Specify the tag given to the command (default: sd-command.yaml)

Reviewer 3: ```suggestion
   -t, --t    Specify the tag given to the command (default: latest)
```

Author: Thanks!

EXAMPLE:
   sd-cmd validate -f ./sd-command.yaml
```

### Promote

> We give the tag specified by `tag` to the version specified by `targetVersion`. The tag will be removed from the version which `tag` is originally assigned to. `targetVersion` can be set exact version or tag (e.g. 1.0.1, latest).

Reviewer: ```suggestion
We give the tag specified by `tag` to the version specified by `targetVersion`. The tag will be removed from the version which `tag` is originally assigned to. `targetVersion` can be set exact version or tag (e.g. 1.0.1, latest).

Giving a `tag` for a `targetVersion` of command. If a `tag` is already set to another version, that tag will be moved to `targetVersion`. `targetVersion` can be set exact version or tag (e.g. 1.0.1, latest).
```

Author: The following is right?
---
Giving a `tag` to a `targetVersion` of command.
---

Reviewer: It may be better.

```bash
USAGE:

>    sd-cmd promote namespace/name targetVersion tag

Reviewer 2: Users will not be able to determine whether it is a fixed string or a variable string.
```suggestion
   sd-cmd promote namespace/name [targetVersion] [tag]
```

EXAMPLE:
   sd-cmd promote foo/bar latest stable
   sd-cmd promote foo/bar 1.0.1 stable
```

### Remove tag

> We remove the tag specified by `tag` from the version which `tag` is assigned to.

Reviewer: ```suggestion
Removing a `tag` from a version of published command.
```
